### DISCUSSION TITLE
NZXT Kraken X73 Coolant Temperature

### DISCUSSION THREAD

**Discussion by Ninemeister** [Rem0o/FanControl.Releases#2119] [Q&A]
Greetings!

A while back I was interested in FanControl being able to reference the coolant temperature on my NZXT X73 AIO. I discovered jonasmalacofilho's liquidctl plugin which did what it was supposed to perfectly, but with a big caveat. I noticed that while the plugin is installed and reading the coolant temperature it causes consistent 5-11MB/s writes to my C: drive. It doesn't sound like much, but since the computer spends a lot of time on that could easily equal over 4TB of writes to the NVMe per week. I posted this as an issue on the liquidctl github over a month ago but still haven't received any responses. 

I suppose my questions are;

1) What is everyone else using to reference NZXT coolant temperatures with FanControl?

2) Does anyone have any insight into why there are the constant 5-11MB/s writes when using the plugin and is there a way to stop them?

Anyone have any insight? Being able to uninstall NZXT CAM could be the final result in making this system as simple as possible.

Thank you!

**Comment by Rem0o** [comment]
Have you taken a look at https://github.com/brokenmass/Fancontrol.NzxtKraken?

Otherwise, LHM has it built-in now I believe, as long as you have the controller section enabled:
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/9a583cae-9038-4de6-892e-43f37a7e87ab)

**Reply by Ninemeister** [reply]
Yes, thank you! This did work for me!
